problemsets-mat223/modules/module12.tex

```diff
@@ -85,8 +85,18 @@
 \begin{example}
 	Let $\mathcal P:\R^2\to\R^2$ be projection onto the $x$-axis and let $\mathcal R:\R^2\to\R^2$ be rotation counter-clockwise
 	by $15^\circ$. Classify each of $\mathcal P$ and $\mathcal R$ as invertible or not.
-
-	XXX Finish
+	
+	Since vectors on y-axis projected onto the x-axis are $\vec 0$, we know
+	\[
+	    \Null(\mathcal P)=\Set*{\vec v\in\R:\vec v=k\mat{0\\1}\text{, for some k}\in\R^2}.
+	\]
+	Therefore, $\Nullity(\mathcal P) \neq 0$, which implies $\mathcal P$ is not invertible by the first property we developed above;\\
+	\\
+	Let $\mathcal R^{-1}: R^2\to\R^2$ be rotation clockwise by $15^\circ$. Therefore, $\mathcal R$ and $\mathcal R^{-1}$ will undo each other. In other words,
```

Comment:
$\mathcal R^{-1}$ is reserved for the inverse. You cannot `Let` it. You can say, `Let $\mathcal Q:\R^2\to\R^2$ be rotation clockwise by $15^\circ$. Then $\mathcal R\circ \mathcal Q=\Id$ and $\mathcal Q\circ \mathcal R=\Id$, and so $\mathcal R$ is invertible with $\mathcal R^{-1}=\mathcal Q$.

Reply:
You still haven't addressed this.